core/scripts/server.ts

```diff
@@ -135,6 +136,43 @@ app.use('/service-worker.js', serve('dist/service-worker.js', false, {
 app.post('/invalidate', invalidateCache)
 app.get('/invalidate', invalidateCache)
 
+if (config.clearCache && config.clearCache.enabled) {
```

Comment:
I suggest moving this code into separate server module (see src/modules/compress for a reference)

Reply:
Did this